src/main/java/picard/arrays/illumina/IlluminaBPMLocusEntry.java

```diff
@@ -0,0 +1,80 @@
+package picard.arrays.illumina;
+
+/**
+ * A simple class to represent a locus entry in an Illumina Bead Pool Manifest (BPM) file
+ */
+public class IlluminaBPMLocusEntry {
+    String ilmnId;              // IlmnID (probe identifier) of locus
```

Comment:
generally prefer comments on separate lines

Reply:
Thanks.  Forgot to get them all